<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>827</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_stop">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始捕获</string>
    </property>
   </widget>
   <widget class="QComboBox" name="interface_choose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>731</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="promiscuous_mode">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>混杂模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="interface_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>接口选择：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filter_write">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>971</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filter_text">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>过滤器：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>1051</width>
      <height>211</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>scr_ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>des_ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>protocol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>length</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>scr_port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>des_port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>info</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>651</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>potocol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>info</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
